bears/generate_package.py

```diff
@@ -113,7 +117,18 @@ def main():
     else:
         bear_version = repr(bear_version) + '.' + str(int(time.time()))
 
-    for bear_file_name in sorted(set(glob('bears/**/*Bear.py'))):
+    local_files = sorted(set(glob('bears/**/*Bear.py')))
+
+    if args.bear:  # Checks if bear name entered is available on the machine
+        for f in local_files:
```

Comment:
`cant you just check whether each of the bears given as arg is in this list?` 
This is what I am doing.
Main thing is, `local_files` will have all the bear files named `somebear.py` . 
So the user will enter `somebear` and not `somebear.py`. 
Hence we can't do it the other way round (i.e check if the arg is in the list. Because name in arg will not exactly match name in list)